instructor/templates/instructor/wb_band_intensity.html

```diff
@@ -0,0 +1,73 @@
+{% extends "instructor/assignment_sidebar.html" %}
+
+{% block right_side %}
+    <div class="scb_ab_s_heading_div">
+        <div class="scb_ab_s_page_title">Band Intensity</div>
+        <div class='scb_ab_s_heading_description'>
+            Define which bands will appear and their relative intensity for each of your samples in your experiment setup.
+        </div>
+
+
+    </div>
+    <div class="scb_ab_s_form_container">
+        <form method="post">{% csrf_token %}
+            {{ formset.management_form }}
+
+            <div class="scb_ab_s_row">
+                <div class="scb_ab_s_col_width_sample">Sample</div>
+                <div class="scb_ab_s_col_width_type_size">Lysate Type</div>
+                <div class="scb_ab_s_col_width_type_size">Size(kDa)</div>
+                <div class="scb_ab_s_col_width_intensity">Relative Band Intensity</div>
+            </div>
+
+            {% for form in formset %}
+                <div class="scb_ab_s_row">
+                    <div class="scb_ab_s_col_width_sample">
+                        {{ forloop.counter }}.
+                        {{ form.instance.strain_protocol.strain.name }},
+                        {{ form.instance.strain_protocol.treatment.drug.name }}
+                        {% if variables.has_concentration %}
+                            ,{{ form.instance.strain_protocol.treatment.drug.concentration }}
```

Comment:
The space before the comma appears because each variable is on a new line. I haven't found a way to get rid of the space without putting everything on one line.